### ISSUE TITLE
I put a Courser of Kruphix on the battlefield with Ashiok, can't play land on top of library

### ISSUE THREAD

**Issue by Zorkony** [magefree/mage#762]
I put a Courser of Kruphix on the battlefield with Ashiok, can't play land on top of library

**Comment by LevelX2** [comment]
I can't reproduce this problem.
You didn't play another land before? Courser does not allow you to playe more lands per turn as normal.

Can you descibe more in detail waht happened?

**Comment by Zorkony** [comment]
I know courser does not allow that, both me and my opponent were confused when I couldn't play the land, neither of us knew why it didn't work. However it is now several games ago so the details are getting blurry. How I recall it, I was playing U/B control, the board was empty except for lands and ashiok, last turn I exiled a courser from my opponent with her, so first thing I do this turn is put it on the battlefield with her minus ability. Subsequently an Urborg was revealed from the top of my deck, but I could not play it (I clicked it multiple times while stack was empty and I had priority etc but nothing happened).
When trying to cast something else from my hand, the game immediately responded after the first click (so it was not stuck or lag or anything).
Maybe I did play a land from my hand and I don't remember it, however I don't see myself being surprised at that moment.. :/
